services/registry/package.json

```diff
@@ -64,15 +64,15 @@
   },
   "scripts": {
     "coverage": "NODE_ENV=test PGUSER=postgres PGDATABASE=entropic_test nyc mocha -R spec",
-    "createdb": "createdbjs -h $PGHOST -U postgres entropic_${NODE_ENV:-dev} || true",
-    "dev": "NODE_ENV=dev PGUSER=postgres PGDATABASE=entropic_${NODE_ENV:-dev} ./server.js",
+    "createdb": "createdbjs -h $PGHOST -U postgres entropic_${NODE_ENV:-development} || true",
+    "dev": "NODE_ENV=development PGUSER=postgres PGDATABASE=entropic_${NODE_ENV:-development} ./server.js",
```

Comment:
Would you mind if we default to `dev`? The postgres database on entropic is set up for that one already.